CrossCompatibility/build.ps1

```diff
@@ -12,7 +12,12 @@ param(
 
 $ErrorActionPreference = 'Stop'
 
-$script:TargetFrameworks = 'netstandard2.0','net451'
+if ($IsWindows -eq $false) {

```

Comment:
This will need to work in PS v3/4, so maybe:

```powershell
if ($IsLinux -or $IsMacOS)
{
    $script:TargetFrameworks = 'netstandard2.0'
}
else
{
    $script:TargetFrameworks = 'netstandard2.0','net451'
}
```

Reply:
This should work on PSv3/v4 which is why I had the code this way.  On PSv3/v4, `$IsWindows` should be `$null` and thus fall into the second block